//package core.io.protocol;

message DceHeartbeat 
{
}
